docs/src/refs.bib

```diff
@@ -26,6 +272,121 @@ @Article{Kem21
   doi           = {10.2307/1989020}
 }
 
+@Article{LS78,
+  author        = {Lascoux, Alain and Sch{\"u}tzenberger, Marcel-Paul},
+  title         = {Sur une conjecture de {H}. {O}. {Foulkes}},
+  zbl           = {0374.20010},
+  journal       = {C. R. Acad. Sci., Paris, S{\'e}r. A},
+  fjournal      = {Comptes Rendus Hebdomadaires des S{\'e}ances de l'Acad{\'e}mie des Sciences, S{\'e}rie A},
+  volume        = {286},
+  pages         = {323--324},
+  year          = {1978},
+  language      = {French},
+  zbmath        = {3582368}
```

Comment:
please check if MathSciNet has DOI

Reply:
This is what MathSciNet has:
```@article {MR472993,
    AUTHOR = {Lascoux, Alain and Sch\"utzenberger, Marcel-Paul},
     TITLE = {Sur une conjecture de {H}. {O}. {F}oulkes},
   JOURNAL = {C. R. Acad. Sci. Paris S\'er. A-B},
  FJOURNAL = {Comptes Rendus Hebdomadaires des S\'eances de l'Acad\'emie des
              Sciences. S\'eries A et B},
    VOLUME = {286},
      YEAR = {1978},
    NUMBER = {7},
     PAGES = {A323--A324},
      ISSN = {0151-0509},
   MRCLASS = {20C30 (14M15)},
  MRNUMBER = {472993},
}
```

I don't know which one is better.

Reply:
Either is fine -- I was mainly hoping for a DOI or URL, but there doesn't seem to be any, so you can leave it as it is.